### ISSUE TITLE
NeuralLabeling: A versatile toolset for labeling vision datasets using Neural Radiance Fields. (arXiv:2309.11966v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#27789]
NeuralLabeling: A versatile toolset for labeling vision datasets using Neural Radiance Fields. (arXiv:2309.11966v1 [cs.CV])<br>
https://ift.tt/Yrh8FKA<br>
<p>We present NeuralLabeling, a labeling approach and toolset for annotating a
scene using either bounding boxes or meshes and generating segmentation masks,
affordance maps, 2D bounding boxes, 3D bounding boxes, 6DOF object poses, depth
maps and object meshes. NeuralLabeling uses Neural Radiance Fields (NeRF) as
renderer, allowing labeling to be performed using 3D spatial tools while
incorporating geometric clues such as occlusions, relying only on images
captured from multiple viewpoints as input. To demonstrate the applicability of
NeuralLabeling to a practical problem in robotics, we added ground truth depth
maps to 30000 frames of transparent object RGB and noisy depth maps of glasses
placed in a dishwasher captured using an RGBD sensor, yielding the
Dishwasher30k dataset. We show that training a simple deep neural network with
supervision using the annotated depth maps yields a higher reconstruction
performance than training with the previously applied weakly supervised
approach.
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
